views/layout.hbs

```diff
@@ -1,100 +1,46 @@
-      })()
-    </script>
-    <![endif]-->
-    <!--[if gte IE 9]><!-->
-      <link href="{{{ assetPath }}}css/fonts.css?0.16.0" media="all" rel="stylesheet" type="text/css" />
-    <!--<![endif]-->
-
-
-    <!--[if lt IE 9]>
-      <script src="{{{ assetPath }}}javascripts/ie.js?0.16.0" type="text/javascript"></script>
-    <![endif]-->
-
-    <link rel="shortcut icon" href="{{{ assetPath }}}images/favicon.ico?0.16.0" type="image/x-icon" />
-
-    <!-- Size for iPad and iPad mini (high resolution) -->
-    <link rel="apple-touch-icon-precomposed" sizes="152x152" href="{{{ assetPath }}}images/apple-touch-icon-152x152.png?0.16.0">
-    <!-- Size for iPhone and iPod touch (high resolution) -->
-    <link rel="apple-touch-icon-precomposed" sizes="120x120" href="{{{ assetPath }}}images/apple-touch-icon-120x120.png?0.16.0">
-    <!-- Size for iPad 2 and iPad mini (standard resolution) -->
-    <link rel="apple-touch-icon-precomposed" sizes="76x76" href="{{{ assetPath }}}images/apple-touch-icon-76x76.png?0.16.0">
-    <!-- Default non-defined size, also used for Android 2.1+ devices -->
-    <link rel="apple-touch-icon-precomposed" href="{{{ assetPath }}}images/apple-touch-icon-60x60.png?0.16.0">
-
-    <meta name="viewport" content="width=device-width, initial-scale=1">
-    <meta property="og:image" content="{{{ assetPath }}}images/opengraph-image.png?0.16.0">
-
-    {{{ head }}}
+    <link href="css/fonts.css?0.16.0" media="all" rel="stylesheet" type="text/css" />
+    <link rel="shortcut icon" href="images/favicon.ico?0.16.0" type="image/x-icon" />
   </head>
 
-  <body class="{{ bodyClasses }}">
-    <script type="text/javascript">document.body.className = ((document.body.className) ? document.body.className + ' js-enabled' : 'js-enabled');</script>
-
-    {{{ bodyStart }}}
-
+  <body class="js-enabled">
+    <noscript>Please enable Javascript in your web browser, in order for this application to function.</noscript>
     <div id="skiplink-container">
       <div>
-        <a href="#content" class="skiplink">{{ skipLinkMessage }}</a>
+        <a href="#content_container" class="skiplink">Click here to skip to content.</a>
       </div>
     </div>
 
     <div id="global-cookie-message">
-
-        {{{ cookieMessage }}}
-
+      This website uses cookies.
     </div>
     <!--end global-cookie-message-->
 
-
-    <header role="banner" id="global-header" class="{{{ headerClass }}}">
+    <header role="banner" id="global-header" class="with-proposition">
       <div class="header-wrapper">
         <div class="header-global">
           <div class="header-logo">
-            <a href="{{{ homepageUrl }}}" title="{{ logoLinkTitle }}" id="logo" class="content">
-              <img src="{{{ assetPath }}}images/gov.uk_logotype_crown_invert_trans.png?0.16.0" width="35" height="31" alt=""> {{{ globalHeaderText }}}
+            <a href="" title="IRC Bed Management" id="logo" class="content">
```

Comment:
For now, I've re-inserted these as hardwired links. Am aware these need to be piped in, to avoid DRY.